From: SeongJae Park <sj@kernel.org>
Date: Mon, 9 Dec 2024 10:21:58 -0800
Subject: [PATCH] mm/damon/paddr: initialize 'applied' of damon_pa_stat2()

The function is using 'applied' variable without initialization.  Do
that.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202412071433.DgqslKRN-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/paddr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/damon/paddr.c b/mm/damon/paddr.c
index 772c6fd790c0..7ac3f55f449a 100644
--- a/mm/damon/paddr.c
+++ b/mm/damon/paddr.c
@@ -479,7 +479,7 @@ static unsigned long damon_pa_migrate(struct damon_region *r, struct damos *s)
 
 static unsigned long damon_pa_stat2(struct damon_region *r, struct damos *s)
 {
-	unsigned long addr, applied;
+	unsigned long addr, applied = 0;
 	LIST_HEAD(folio_list);
 
 	for (addr = r->ar.start; addr < r->ar.end; addr += PAGE_SIZE) {
-- 
2.39.5

